src/styles/components/milestones.scss

```diff
@@ -0,0 +1,110 @@
+.milestones-wrapper {
+  overflow: auto;
+
+  &::-webkit-scrollbar {
+    width: 0px;
+    background: transparent;
+  }
+
+  .milestones {
+    display: inline-block;
+    position: relative;
+    padding: 24px 0;
+    list-style: none;
+
+    &:before {
+      position: absolute;
+      content: " ";
+      height: 3px;
+      top: auto;
+      bottom: 30px;
+      left: 56px;
+      right: 0;
+      width: 100%;
+      margin-bottom: 20px;
+      margin-left: 8px;
+      background-color: #c4c4c4;
```

Comment:
this could be a variable

Reply:
Updated this to match existing gray. `$gray-3` to be precise.